rpm2extents.c

```diff
@@ -261,7 +277,7 @@ static rpmRC process_package(FD_t fdi, FD_t digestori, FD_t validationi)
     uint32_t offset_ix = 0;
     size_t len;
     int next = 0;
-	struct rpmlead_s l;
+    struct rpmlead_s l;
```

Comment:
not that we may have properly abode to the [RPM coding style](https://github.com/rpm-software-management/rpm/blob/master/CONTRIBUTING.md#coding-style), but I suspect your text editor may need to be properly set.